test/packages/EATest/CMakeLists.txt

```diff
@@ -19,10 +19,19 @@ add_definitions(-D_CRT_SECURE_NO_WARNINGS)
 #-------------------------------------------------------------------------------------------
 # Compiler Flags
 #-------------------------------------------------------------------------------------------
-set(CMAKE_CXX_STANDARD 11)
-if(UNIX)
-    SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+if( UNIX AND "${CMAKE_CXX_COMPILER_ID}" STREQUAL "Intel" )
+  set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -fasm-blocks" )
 endif()
+
+if (CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+	SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
+endif()
+
+if (CMAKE_CXX_COMPILER_ID MATCHES "Clang" OR CMAKE_CXX_COMPILER_ID MATCHES "GNU")
+	set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -D_DEBUG")
```

Comment:
Just checked, _DEBUG is correctly picked up by Qt Creator and EASTL_DEBUG functions as expected. If this is not picked up correctly by other IDEs, I suppose we could make the change.